package.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "new",
-  "version": "1.0.0",
-  "description": "Fetching posts from instagram.",
-  "main": "index.js",
   "author": "shynox",
-  "license": "MIT",
   "dependencies": {
     "commander": "^8.3.0",
     "fs": "^0.0.1-security",

```

Comment:
Remove fs from here. It's built-in with nodejs.

Reply:
> 

edited to match the lesser character